diff -Naur a/setup.py b/setup.py
--- a/setup.py	2020-09-15 22:05:23.481544264 +0530
+++ b/setup.py	2020-09-15 22:03:24.794947000 +0530
@@ -40,7 +40,6 @@
 # Key Values that Change Each Release
 ###############################################################################
 SETUP_REQUIREMENTS = {'numpy': '1.15',  # released June 2018
-                      'scipy': '1.1',  # released May 2018
                       }
 
 REQ_NOT_MET_MSG = """
